src/constant/routes.ts

```diff
@@ -12,18 +12,23 @@ const routes = {
   },
   shiksha: '/shiksha',
   shikshaExplore: '/shiksha/explore',
+  shikshaMycu: '/shiksha/courses',
+  shikshaCourses: '/shiksha/my-courses',
+
   allCourses: {
     logicBuildingForEveryone: '/shiksha/logic-building-for-everyone',
     basicsOfProgrammingWithJS: '/shiksha/basics-of-programming-with-js',
   },
   interviewPrep: '/interview-prep',
   interviewPrepExplore: '/interview-prep/explore',
+
   workshops: '/workshops',
   os: '/os',
   contactUs: '/contact',
   internals: {
     landing: {
       products: 'products',
+      shiksha_courses: 'shiksha_courses',
```

Comment:
Not sure if needed. Even if needed then follow the same code style. write like shikshaCourses.